tendrl/gluster_integration/manager/rpc.py

```diff
@@ -159,6 +159,9 @@ def _run(self):
             try:
                 LOG.info("%s run..." % self.__class__.__name__)
                 self._server.run()
+            except etcd.EtcdKeyNotFound as ex:
+                LOG.info("Given key is not present in etcd . %s", ex)
```

Comment:
can you have it as a debug level log, as it is logged every 3 sec once and the queue is expected to be empty until the first job is submitted.